apps/web/src/components/templates/email-editor/EmailContentCard.tsx

```diff
@@ -79,6 +80,7 @@ export function EmailContentCard({
   return (
     <>
       {!isIntegrationActive ? <LackIntegrationError channelType="E-Mail" /> : null}
+      <VariableManager template={`steps.${index}.template`} />
```

Comment:
Maybe in the meanwhile let's put it under each content area and not above? Will consult with the designer about this one, but for now I think it can be below. Wdyt?